deployment/LogAnalytics.md

```diff
@@ -27,23 +27,14 @@ Additionally, the OMS Agent must be installed on the cluster.
 ## Install OMS Agent onto Cluster
 1. Navigate to your OMS Workspace in the portal by finding it under your "Log Analytics" resources
 2. Find your Workspace Key by clicking on the Advanced Settings blade, then Connected Sources > Linux Servers
-3. Copy the primary key and paste it into the helm command in step 5
-4. Open a terminal and run this command to find the Workspace ID:
+3. Copy the primary key and paste it into line 19 of [installOmsAgent.sh](/deployment/installOmsAgent.sh)
+4. Run `installOmsAgent.sh`
 
-```
-WSID=$(az resource show --resource-group resourcegroupname  --resource-type Microsoft.OperationalInsights/workspaces --name loganalyticsworkspacename | grep customerId | sed -e 's/.*://')
-```
-5. Run the following with your proper Workspace Key:
-
-```
-helm install --name omsagent --set omsagent.secret.wsid=$WSID --set omsagent.secret.key=<key-value> stable/msoms
-```
 ## Add solutions to your Log Analytics workspace:
-1. If you haven't already done so, sign in to the Azure portal using your Azure subscription.
-2. Select Create a resource > Monitoring + Management > See all.
-3. To the right of Management Solutions, click More.
-4. In the Management Solutions blade, select a management solution that you want to add to a workspace.
-5. In the management solution blade, review information about the management solution, and then click Create.
-6. In the management solution name blade, select a workspace that you want to associate with the management solution.
-7. Optionally, change workspace settings for the Azure subscription, resource group, and location. You can also choose Automation options. Click Create.
+1. In the Azure portal, select Create a resource > Monitoring + Management > See all.
```

Comment:
Is it up to date? I don't see "Monitoring + Management" when I click "Create a resource".